composer.json

```diff
@@ -87,7 +87,8 @@
         "symfony/web-profiler-bundle": "^4.2 || ^5.0",
         "symfony/yaml": "^3.4 || ^4.0 || ^5.0",
         "twig/twig": "^1.42.3 || ^2.12",
-        "webonyx/graphql-php": ">=0.13.1 <1.0"
+        "webonyx/graphql-php": ">=0.13.1 <1.0",
+        "zz-api-platform-dev/stubs-mongodb": "@dev"
```

Comment:
what's with the zz-api-platform-dev ? :p

Reply:
So that it's sorted at the end. 🤯